media/css/security/components/level.scss

```
@@ -25,7 +25,7 @@ $image-path: '/media/protocol/img';
     font-weight: bold;
     line-height: 1;
     padding: 4px 2px 4px 4px;
```

```suggestion
    padding: 4px;
```
I think this would also be a good addition. 🙂